jupyter_accessibility/jupyter-axe/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "@jupyterlab/a11ytests",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Run automated accessibility checks against JupyterLab",
+  "license": "BSD-3-Clause",
+  "author": "Project Jupyter",
+  "scripts": {
+    "start": "jupyter lab --config ./jupyter_server_config.py",
+    "test": "playwright test",
+    "test:debug": "PWDEBUG=1 playwright test"
+  },
+  "dependencies": {
+    "@jupyterlab/galata": "4.0.2",
+    "expect-axe-playwright": "gabalafou/expect-axe-playwright#f8af64b"
```

Comment:
This has to be fixed. 

I forked expect-axe-playwright because its API wasn't flexible enough for what we want to do.

I see two possible paths forward:

- Upstream my changes to expect-axe-playwright (will probably end up taking on a new form)
- Copy-paste whatever code is useful from expect-axe-playwright into our own source code in this repo and thereby get rid of the dependency altogether. 

I'm not sure which approach is best. I lean slightly towards the second option simply because these additional libraries like expect-axe-playwright are very slim. I'm not sure they bring enough to the table to outweigh the downsides to being dependent on them. The only dependency where I don't want to reinvent the wheel is `axe-reporter-html`, which provides a nice HTML template for summarizing the axe-core violations.

Reply:
so the changes you made in your fork are to export the output files?

other options:
* could we include our patched version of axe fixture in this library itself, and ship it as part of the a package?
* is this something we'd really want to target in `galata`?